### DISCUSSION TITLE
Add support for Upsert mutations

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#6067] [Feature requests]
Will be good to add Upsert mutations, Prisma already have support for it:
https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert

**Comment by bladey** [comment]
Thanks @MurzNN! the survey results also indicated strong support for this, we'll take it into account.

**Comment by mariomnts** [comment]
I think this is a duplicate of https://github.com/keystonejs/keystone/discussions/8117

**Reply by dcousens** [reply]
Indeed! Locking this discussion, please see https://github.com/keystonejs/keystone/discussions/8117
